package.json

```diff
@@ -31,6 +31,7 @@
     "typescript": "^3.6.4",
     "vinyl": "~2.0.1",
     "vinyl-fs": "^3.0.2",
+    "vue-template-compiler": "^2.6.11",
```

Comment:
I'm questioning whether we should have this be a `devDependency` and require people who rely on Vue to install it to use the plugin. I'm not keen on adding many dependencies to the project. I especially doubt Vue to be used by most.

wdyt @vincaslt

Reply:
Not sure. At least add it to peer dependencies then?

Reply:
vue-template-compiler will definitely be in the project on vuejs
but if you need to parse files outside the project, the user will have to put this package himself.